src/libs/actions/Session/index.js

```diff
@@ -448,11 +448,12 @@ function setOrChangePassword(accountID, validateCode, password, authToken) {
  * @param {String} validateCode
  * @param {String} login
  * @param {String} authToken
+ * @return {Promise}
  */
 function validateEmail(accountID, validateCode) {
     Onyx.merge(ONYXKEYS.USER_SIGN_UP, {isValidating: true});
     Onyx.merge(ONYXKEYS.SESSION, {error: ''});
-    API.ValidateEmail({
+    return API.ValidateEmail({
```

Comment:
Where is this promise used? I can't find it. Again, please never return a promise from an action method unless it's absolutely necessary for another action.